op-guide/tune-tikv.md

```diff
@@ -26,21 +26,233 @@ Each CF also has a separate `write buffer`. You can configure the size by settin
 
 ## Parameter specification
 
-[TiKV Configuration](https://github.com/pingcap/tikv/blob/master/etc/config-template.toml).
+```
+# Log level: trace, debug, info, warn, error, off.
+log-level = "info"
 
-## TiKV memory usage
+[server]
+# Set listening address
+# addr = "127.0.0.1:20160"
+
+# It is recommended to use the default value.
+# notify-capacity = 40960
+# messages-per-tick = 4096
+
+# Size of thread pool for gRPC
+# grpc-concurrency = 4
+# The number of gRPC connections between each TiKV instance
+# grpc-raft-conn-num = 10
+
+# Most read requests from TiDB are sent to the coprocessor of TiKV. This parameter is used to set the number of threads 
+# of the coprocessor. If there are many read requests for the business, the number of the threads of the coprocessor 
+# will be increased. But this number should be smaller than that of CPU cores of the system. For example, there are 
```

Comment:
Suggested version: 
If many read requests exist, add the number of threads and keep the number within that of the system CPU cores.